nodeup/pkg/model/kubelet.go

```diff
@@ -145,6 +147,17 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 		flags += " --network-plugin-dir=" + b.CNIBinDir()
 	}
 
+	// If a cluster has a container registry set, flag kubelet to use the pause container from that registry.
+	if b.Cluster.Spec.Assets != nil && b.Cluster.Spec.Assets.ContainerRegistry != nil {
+		// TODO another hardcoded version
+		pause, err := assets.GetGoogleImageRegistryContainer(&b.Cluster.Spec, "pause-amd64:3.0")
+
```

Comment:
Can you set your IDE not to insert this blank line?  It's really unintuitive, to _my_ mental model at least.